### PR TITLE
First draft changes

### PR DESCRIPTION
**Pull request by jasonwolfganggenesys**

### CODE REVIEW

File: Blueprint/index.md [hunk 1]
```diff
@@ -0,0 +1,241 @@
+---
+title: Update the presence of a Genesys Cloud User from Microsoft Teams User Presence Update
+author: yuri.yeti
```
Comment (AgnesCorpuz) [inline Blueprint/index.md]: You may add yourself as the author of this blueprint. Please change the value to "jason.wolfgang".

File: Blueprint/index.md [hunk 2]
```diff
@@ -0,0 +1,241 @@
...
+category: 6
+summary: |
+  This Genesys Cloud Developer Blueprint describes how to set up Genesys Cloud to update a Genesys Cloud user's presence when the corresponding Microsoft Teams user has a presence change.
+---
+This Genesys Cloud Developer Blueprint describes how to set up Genesys Cloud to update a Genesys Cloud user's presence when the corresponding Microsoft Teams user has a presence change.
+
+The following illustration shows the end-to-end user experience that this solution enables.
+
+![End-to-end user experience](images/MS-Teams-Updates-GC.gif "End-to-end user experience")
+
```
Comment (AgnesCorpuz) [inline Blueprint/index.md:18]: Please also add a flowchart of the solution.

**Review by AgnesCorpuz — CHANGES_REQUESTED** [review]
Overall looks good. I've added some in-line comments. Our team will add to this blueprint to have a terraform section. But we can go ahead and publish this as is. 

A couple more notes:
- Can we let the readers know that it is still in draft mode by adding a note in the README file and that they should expect some updates on the repo.
- Please rename the "Blueprint" folder to "blueprint" - not to be over critical but I'm just not sure if our deployment app is case sensitive and it might not get picked up.
- Create an "exports" folder and put all importable files inside.

**Comment by jasonwolfganggenesys** [comment]
> Overall looks good. I've added some in-line comments. Our team will add to this blueprint to have a terraform section. But we can go ahead and publish this as is.
> 
> A couple more notes:
> 
>     * Can we let the readers know that it is still in draft mode by adding a note in the README file and that they should expect some updates on the repo.
> 
>     * Please rename the "Blueprint" folder to "blueprint" - not to be over critical but I'm just not sure if our deployment app is case sensitive and it might not get picked up.
> 
>     * Create an "exports" folder and put all importable files inside.

These all make sense!  Please find the updates reflected in the last commit to this PR.

**Pull request merged**